### DISCUSSION TITLE
Is the diagram in the README file generated by this module?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [go-jet/jet#293] [Q&A]
I want to generate some db diagram base on the code generated by jet.
But I saw there's already a nice diagram in the README,
so want to ask if it's generate by jet?

**Comment by go-jet** [comment]
The db diagram is generated with dbeaver.

**Reply by LintaoAmons** [reply]
okie， I see. The community version don't have the edit functionality, sad.
